src/Grpc/test/InteropTests/InteropTests.cs

```diff
@@ -25,9 +25,26 @@ public InteropTests(ITestOutputHelper output)
             _output = output;
         }
 
-        [Theory]
-        [QuarantinedTest("https://github.com/dotnet/aspnetcore/issues/22101")]
-        [MemberData(nameof(TestCaseData))]
+        // All interop test cases, minus GCE authentication specific tests
+        [ConditionalTheory]
+        [InlineData("empty_unary")]
+        [InlineData("large_unary")]
+        [InlineData("client_streaming")]
+        [InlineData("server_streaming")]
+        [InlineData("ping_pong", Skip = "https://github.com/dotnet/aspnetcore/issues/22101")]
+        [InlineData("empty_stream")]
+        [InlineData("cancel_after_begin")]
+        [InlineData("cancel_after_first_response")]
+        [InlineData("timeout_on_sleeping_server")]
+        [InlineData("custom_metadata")]
+        [InlineData("status_code_and_message")]
+        [InlineData("special_status_message")]
+        [InlineData("unimplemented_service")]
+        [InlineData("unimplemented_method")]
+        [InlineData("client_compressed_unary")]
+        [InlineData("client_compressed_streaming")]
+        [InlineData("server_compressed_unary")]
+        [InlineData("server_compressed_streaming")]
```

Comment:
Skipping a test isn't the same thing as quarantining it. We lose _all_ data about whether a fix has worked and can't test it out for 30 days with this approach.

The canonical approach would be to make `InteropTestCase(...)` `private` and add individual `[Fact]`s that call it (and are `public`.